android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -15,9 +17,8 @@ import com.bitmovin.player.api.ui.PlayerViewConfig
 import com.bitmovin.player.reactnative.converter.JsonConverter
 import com.bitmovin.player.reactnative.ui.RNPictureInPictureDelegate
 import com.bitmovin.player.reactnative.ui.RNPictureInPictureHandler
-import com.facebook.react.bridge.LifecycleEventListener
-import com.facebook.react.bridge.ReactApplicationContext
-import com.facebook.react.bridge.ReactContext
+import com.facebook.react.ReactActivity
+import com.facebook.react.bridge.*
```

Comment:
I am surprised or linitng doesn't flag * imports. 🤔  Maybe something to consider for the future. Fine for me for now as linter configuration is out of scope for the issue.

Reply:
We have a linting rule that allows wildcard imports for react bridge because in a lot of files there would be really a lot react bridge imports.